documentation/src/test/java17/org/hibernate/search/documentation/mapper/orm/projection/Author.java

```diff
@@ -29,10 +31,18 @@ public class Author {
 	@FullTextField(analyzer = "name", projectable = Projectable.YES)
 	private String lastName;
 
+	@VectorField(dimension = 4)
```

Comment:
please ignore this test 😃 I only have it to test an annotation with real db and backend at the same time, I'll have a proper test for the docs once I add a section for a new predicate 🙈